app/code/Magento/Fedex/Model/Carrier.php

```diff
@@ -1571,7 +1571,7 @@ private function processTrackingDetails(\stdClass $trackInfo)
         ];
 
         if (!empty($trackInfo->ShipTimestamp)) {
-            $datetime = \DateTime::createFromFormat(\DateTime::ISO8601, $trackInfo->ShipTimestamp);
+            $datetime = \DateTime::createFromFormat('U', strtotime($trackInfo->ShipTimestamp));
```

Comment:
**UPD** FedEx support has confirmed, if date time doesn't contain timezone, when date time in UTC.